### PR TITLE
Seed data

### PR DESCRIPTION
**Pull request by trongnguyennguyen**

### CODE REVIEW

**Comment by trongnguyennguyen** [comment]
ready

**Comment by trangkarol** [comment]
Commented
E rebase về 1 commit e nhé

**Comment by trongnguyennguyen** [comment]
ready

**Comment by trangkarol** [comment]
Commented
Chia làm 3 pull e nhé

**Comment by trongnguyennguyen** [comment]
ready

File: app/Models/Book.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    protected $fillable = [
+        'name',
+        'author',
+        'number_page',
+        'description',
+        'category_id',
+        'status'
+    ];
+
+    public function like()
```
Comment (trangkarol) [inline app/Models/Book.php]: Nếu là hasMany() thì name function là sự nhiều.  Vd: likes()

File: app/Models/Book.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        return $this->hasMany(Like::class);
+    }
+
+    public function listrequest()
```
Comment (trangkarol) [inline app/Models/Book.php]: Tên function có dangj cameCase. Vd: listRequest()

File: app/Models/Book.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        return $this->hasMany(Review::class);
+    }
+
+    public function picture()
+    {
+        return $this->hasMany(Picture::class);
+    }
+
+    public function categorie()
```
Comment (trangkarol) [inline app/Models/Book.php]: category()

File: app/Models/Activity.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
```
Comment (trangkarol) [inline app/Models/Activity.php]: Đưa các file mode vào folder Models e nhé

**Comment by trangkarol** [comment]
Commented
Quá nhiều file

**Comment by trongnguyennguyen** [comment]
ready

**Comment by trongnguyennguyen** [comment]
ready

**Comment by trongnguyennguyen** [comment]
ready

File: database/seeds/ActivitiesTableSeeder.php [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+            'user_id' => '1',
+            'book_id' => '375',
+            'action' => str_random(50)
+        ]);
+
+        DB::table('activities')->insert([
+            'user_id' => '2',
+            'book_id' => '245',
+            'action' => str_random(50)
+        ]);
```
Comment (trangkarol) [inline database/seeds/ActivitiesTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

**Comment by trangkarol** [comment]
commented

**Comment by trongnguyennguyen** [comment]
ready

File: database/seeds/ActivitiesTableSeeder.php (hunk 5 above)
Comment (trangkarol) [reply database/seeds/ActivitiesTableSeeder.php]: fix chỗ này e ơi

File: database/seeds/BookTabelSeeder.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+            'status' => 'read',
+        ]);
+
+        DB::table('books')->insert([
+            'name' => "Lập trình Ruby",
+            'author' => 'Framgia',
+            'page_number' => '250',
+            'description' => 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sapiente esse necessitatibus neque',
+            'category_id' => '1',
+            'status' => 'unread',
```
Comment (trangkarol) [inline database/seeds/BookTabelSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/CategoriesTableSeeder.php [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    public function run()
+    {
+        DB::table('categories')->insert([
+            'parent_id' => '1',
+            'name' => 'Lập trình'
+        ]);
+
+        DB::table('categories')->insert([
+            'parent_id' => '2',
+            'name' => 'Bách khoa toàn thư'
```
Comment (trangkarol) [inline database/seeds/CategoriesTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/CommentTableSeeder.php [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+            'review_id' => '14',
+            'user_id' => '1',
+            'content' => 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sapiente esse necessitatibus neque',
+        ]);
+
+        DB::table('comments')->insert([
+            'review_id' => '10',
+            'user_id' => '2',
+            'content' => 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sapiente esse necessitatibus neque',
+        ]);
```
Comment (trangkarol) [inline database/seeds/CommentTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/DatabaseSeeder.php [hunk 9]
```diff
@@ -11,6 +12,17 @@ class DatabaseSeeder extends Seeder
...
+        ActivitiesTableSeeder::class,
+        BookTableSeeder::class,
+        CategoriesTableSeeder::class,
+        FollowingTableSeeder::class,
+        LikeTableSeeder::class,
+        ListRequestTable::class,
+        PictureTableSeeder::class,
+        RequestTableSeeder::class,
+        ReviewTableSeeder::class,
+    ]);
```
Comment (trangkarol) [inline database/seeds/DatabaseSeeder.php]: Indent: 4 space

File: database/seeds/FollowingTableSeeder.php [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        DB::table('followings')->insert([
+            'user_id' => '1',
+            'follow_id' => '17'
+        ]);
+
+        DB::table('followings')->insert([
+            'user_id' => '2',
+            'follow_id' => '12'
+        ]);
```
Comment (trangkarol) [inline database/seeds/FollowingTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/LikeTableSeeder.php [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        DB::table('likes')->insert([
+            'user_id' => '1',
+            'book_id' => '375'
+        ]);
+
+        DB::table('likes')->insert([
+            'user_id' => '2',
+            'book_id' => '245'
+        ]);
```
Comment (trangkarol) [inline database/seeds/LikeTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/ListRequestTableSeeder.php [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+            'pay_day' => '2018-12-22'
+        ]);
+
+        DB::table('list_requests')->insert([
+            'user_id' => '2',
+            'book_id' => '245',
+            'name' => 'Lập trình Ruby',
+            'borrowed_day' => '2018-12-12',
+            'pay_day' => '2018-12-13'
+        ]);
```
Comment (trangkarol) [inline database/seeds/ListRequestTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/PictureTableSeeder.php [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        DB::table('pictures')->insert([
+            'book_id' => '375',
+            'url' => 'http://hinhanhdep.pro/content/uploads/2016/03/hinh-anh-hai-huoc-6.jpg'
+        ]);
+
+        DB::table('pictures')->insert([
+            'book_id' => '245',
+            'url' => 'http://hinhanhdep.pro/content/uploads/2016/03/hinh-anh-hai-huoc-6.jpg'
+        ]);
```
Comment (trangkarol) [inline database/seeds/PictureTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/RequestTableSeeder.php [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+            'user_id' => '1',
+            'book_name' => 'Lập trình Laravel',
+            'status' => str_random(50)
+        ]);
+
+        DB::table('requests')->insert([
+            'user_id' => '2',
+            'book_name' => 'Lập trình Ruby',
+            'status' => str_random(50)
+        ]);
```
Comment (trangkarol) [inline database/seeds/RequestTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/ReviewTableSeeder.php [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+            'book_id' => '357',
+            'user_id' => '1',
+            'content' => 'hay'
+        ]);
+
+        DB::table('reviews')->insert([
+            'book_id' => '245',
+            'user_id' => '2',
+            'content' => 'không hay'
+        ]);
```
Comment (trangkarol) [inline database/seeds/ReviewTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: database/seeds/UsersTableSeeder.php [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+            'level' => '0'
+        ]);
+
+        DB::table('users')->insert([
+            'name' => 'phi',
+            'email' => 'phitest@gmail.com',
+            'password' => bcrypt('12345678'),
+            'phone' => str_random(10),
+            'address' => str_random(10),
+            'level' => '1'
```
Comment (trangkarol) [inline database/seeds/UsersTableSeeder.php]: nghiên cứu dùng https://laravel.com/docs/5.5/seeding#using-model-factories

File: resources/views/home.blade.php [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+                <div class="panel-heading">Dashboard</div>
+
+                <div class="panel-body">
+                    @if (session('status'))
+                        <div class="alert alert-success">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    You are logged in!
```
Comment (trangkarol) [inline resources/views/home.blade.php]: I18n

**Comment by trangkarol** [comment]
commented

**Comment by trongnguyennguyen** [comment]
ready

**Pull request merged**